doc/index.md

```diff
@@ -157,6 +157,7 @@ self
 getting_started/index
 how_to/index
 gallery/index
+background/index
```

Comment:
> * Add `Background` to the top-level TOC


I think it already is, no?

@philippjfr, are you referring to something else?

Reply:
Missed it sorry!